src/Sulu/Component/Content/Types/Rlp/Strategy/RlpStrategy.php

```diff
@@ -270,12 +270,7 @@ public function loadByResourceLocator($resourceLocator, $webspaceKey, $languageC
      */
     public function isValid($path, $webspaceKey, $languageCode, $segmentKey = null)
     {
-        return $path !== '/' && $this->cleaner->validate($path) && $this->mapper->unique(
```

Comment:
i could also leave this function as it is and use the `$this->cleaner->validate()` directly.